var/spack/repos/builtin/packages/openssh/package.py

```diff
@@ -61,3 +63,33 @@ def configure_args(self):
         # install step and fail if they cannot do so.
         args = ['--with-privsep-path={0}'.format(self.prefix.var.empty)]
         return args
+
+    def install(self, spec, prefix):
+        make('install', parallel=False)
```

Comment:
It'd be good to add a small comment with justification (I know it's in the commit message too)

Reply:
Good catch, done!